src/com/owncloud/android/operations/SynchronizeFolderOperation.java

```diff
@@ -379,7 +379,7 @@ private void mergeRemoteFolder(ArrayList<Object> folderAndFiles)
     }
 
 
-    private void preparePushOfLocalChanges() throws OperationCancelledException {
+    private void preparePushOfLocalChanges() {
```

Comment:
This exception is being caught at https://github.com/owncloud/android/blob/eabc46ad9999263cb9f70137b477192e4ba26500/src/com/owncloud/android/operations/SynchronizeFolderOperation.java#L228 , why should it be deleted if could still appear?

Reply:
Well this catch is never performet. I checked weather there is a methode inside `preparePushOfLocalChanges()` that throws `SynchronizeFolderOperation.java` but there is non.